apps/documentation/src/app/using-smart-ng-rx/web-sockets/index.md

```diff
@@ -0,0 +1,100 @@
+# WebSockets
+
+To use SmartNgRX with WebSockets, you'll need to establish your own WebSocket connection. The only information the that you'll need to pass up to the client are which type of action needs to be performed, the table name that is impacted, and the ids of the records that are impacted.
+
+## Updates
+
+So, as an example, if you update the content of a row in the `users` table, you'll need to send a message to the client that might like this:
+
+```json
+{
+  "action": "update",
+  "table": "users",
+  "ids": ["1"]
+}
+```
+
+Normally, there will only be one element in the array, but we've created the APIs to handle multiple IDs in case you need to update multiple records at once.
+
+Then you would pass this information to SmartNgRX using this code:
+
+```typescript
+import { handleSocketNotification } from '@smart/smart-ngrx/socket/handle-socket-notification.function';
+
+handleSocketNotification(data.table, data.action, data.ids);
+```
+
+## Creates
+
+When you add a row to the `users` table, you might think that you'd send a message the might look like this:
```

Comment:
**suggestion (documentation):** Correct 'a message the might look like this' to 'a message that might look like this'